src/status_im2/contexts/chat/messages/list/events.cljs

```diff
@@ -5,6 +5,17 @@
     [utils.re-frame :as rf]
     [utils.red-black-tree :as red-black-tree]))
 
+(defn- is-system-message?
```

Comment:
sets are functions, as they implement `iFn`. https://clojure.org/reference/data_structures#Sets 

So you can just do

`(def is-system-message? #{constants/....})`

and it can be used in the same way you are using it below:

`(is-system-message? content-type)`

Reply:
Aha, that makes sense. 🙏